helm/robusta/templates/auth-config.yaml

```diff
@@ -3,6 +3,7 @@ apiVersion: v1
 kind: Secret
 metadata:
   name: robusta-auth-config-secret
+  namespace: {{ .Release.Namespace }}
```

Comment:
Does .Release.Namespace always has a value?
It's either the current namespace, or the one you pass to the helm command?

Reply:
from my test its set to default when i dont use  something.